ruby/facter

```diff
@@ -1 +1 @@
-Subproject commit 8ff4fa5e0251aaa1b3fea1e2e308a34b2ffe6027
+Subproject commit 691e2b164ec4144d0ce80c14d3132e89242f26cc
```

Comment:
Facter 2.4.3 is what is in the 1.0.z puppet-agent versions, the latest available at the moment, so yes, that makes sense to me.